<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LookWindow</class>
 <widget class="QWidget" name="LookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#0facf1;&quot;&gt; 监   控   界   面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>79</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sotpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>灯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openLedButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeLedButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>风扇</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openFsButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>170</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeFsButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>返      回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>当前温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>℃</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>当前湿度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>HR</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tempBrowser">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="humBrowser">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>光照强度</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="luxBrowser">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lux</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="led">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="fan">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>蜂鸣器</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="beep">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openbeeButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closebeeButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>startButton</zorder>
  <zorder>sotpButton</zorder>
  <zorder>label_3</zorder>
  <zorder>openLedButton</zorder>
  <zorder>closeLedButton</zorder>
  <zorder>label_4</zorder>
  <zorder>openFsButton</zorder>
  <zorder>closeFsButton</zorder>
  <zorder>returnButton</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>humBrowser</zorder>
  <zorder>label_9</zorder>
  <zorder>luxBrowser</zorder>
  <zorder>label_10</zorder>
  <zorder>tempBrowser</zorder>
  <zorder>led</zorder>
  <zorder>fan</zorder>
  <zorder>label_11</zorder>
  <zorder>beep</zorder>
  <zorder>openbeeButton</zorder>
  <zorder>closebeeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
